packages/core/src/utils.js

```diff
@@ -662,21 +672,28 @@ export function stubExistingAdditionalProperties(
   return schema;
 }
 
-export function resolveSchema(schema, rootSchema = {}, formData = {}) {
+export const resolveSchema = _.memoize(_resolveSchema, cacheKeyFn);
+
+export function _resolveSchema(
+  schema,
+  rootSchema = DEFAULT_ROOT_SCHEMA,
+  formData = DEFAULT_FORM_DATA
+) {
   if (schema.hasOwnProperty("$ref")) {
     return resolveReference(schema, rootSchema, formData);
   } else if (schema.hasOwnProperty("dependencies")) {
     const resolvedSchema = resolveDependencies(schema, rootSchema, formData);
     return retrieveSchema(resolvedSchema, rootSchema, formData);
-  } else if (schema.hasOwnProperty("allOf")) {
+  } else if (schema["allOf"]) {
```

Comment:
what's the reason for this change?

Reply:
`allOf` may be set to undefined here: https://github.com/nickgros/react-jsonschema-form/blob/5e7a08dce80c0c1c85293ac3465c63d65693c6d4/packages/core/src/utils.js#L767

I think I changed that per a comment on the previous PR. Could use `delete` instead and revert this.

Reply:
Switching that line to `delete` actually fixed the `liveOmit` issue that I mentioned [here](https://github.com/rjsf-team/react-jsonschema-form/pull/2466#issuecomment-884929527). I'm not sure if it fixed the stack overflow error that you saw.